src/cc/libbpf.c

```diff
@@ -949,7 +949,7 @@ static int create_probe_event(char *buf, const char *ev_name,
                               const char *config1, uint64_t offset,
```

Comment:
the root issue here is that config1 already contains a path that is within the target process's mount namespace since #2324